src/test/resources/features/account.feature

```diff
@@ -24,13 +31,35 @@ Feature: Requests for Account endpoint
   @UpdateAccount
   Scenario Outline: Update a Account
     Given I build the pathParams and updated body to request
```

Comment:
"I set the pathParams and the updated body for the request"

Reply:
Good suggestion. I will change it.